### PR TITLE
Add new Volunteer AaryaChopkar

### PR DESCRIPTION
**Pull request by accodes21**
Added new Volunteer.
Code matches existing pattern.

### CODE REVIEW

**Review by codewithvoid — APPROVED** [review]

**Comment by codewithvoid** [comment]
Perfect. Thanks Aarya!

File: data/users/AaryaChopkar.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (codewithvoid) [inline data/users/AaryaChopkar.json:1]: filename should be github username .json so `accodes21.json` for you

File: data/users/AaryaChopkar.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      },
+      {
+        "slug": "programming",
+        "topics": ["python"]
+      }
+    ],
+    "contacts": {
+      "twitter": "ChopkarAarya"
+    }
+  }
```
Comment (RakeshSangem) [inline data/users/AaryaChopkar.json:18]: Could you align the brace here!

Comment (codewithvoid) [reply data/users/AaryaChopkar.json:18]: Right use https://jsoneditoronline.org

**Review by RakeshSangem — APPROVED** [review]
Great Job👏

**Pull request merged**